services/sendemail.js

```diff
@@ -0,0 +1,30 @@
+const nodemailer = require("nodemailer");
+
+exports.sendEmail = async (email) => {
```

Comment:
this is good, But we should be able to use this for all types of emails. Make it so that it reusable. You can create new folder with email templates and then If i call this service function from for example login service I just pass email template and everything works fine